lib/virtualGraphs.js

```diff
@@ -18,12 +18,13 @@ const add = (conn, name, mappings, options) => {
       name,
       mappings,
       options,
+      db: params.db || null,
```

Comment:
I'm not sure if the null is possible, perhaps both 'null' and 'undefined' are converted to 'null' in Java?

Reply:
`JSON.stringify` will actually leave out `undefined` values entirely. So, I think the null is unnecessary (same for the instance below this one).